languagetool-core/src/main/java/org/languagetool/AnalyzedSentence.java

```diff
@@ -309,7 +322,29 @@ public String getAnnotations() {
    * @since 2.5
    */
   public Set<String> getLemmaSet() {
-    return lemmaSet;
+    return lemmaOffsets.keySet();
+  }
+
+  /**
+   * @return all offsets in {@link #getTokensWithoutWhitespace()} where tokens with the given text occur (case-insensitive),
+   * or {@code null} if there are no such occurrences
+   * @since 5.2
```

Comment:
This (and below) probably should be changed to 5.3

Reply:
Thanks, fixed